libtransact/src/protocol/receipt.rs

```diff
@@ -420,37 +461,37 @@ impl std::fmt::Display for TransactionReceiptBuilderError {
 }
 
 #[derive(Default, Clone)]
-pub struct TransactionReceiptBuilder {
+pub struct ValidTransactionReceiptBuilder {
     pub state_changes: Vec<StateChange>,
```

Comment:
Not sure I understand why these are all public

Reply:
Updated